[blog] 2021 (#30425)
mui-org/material-ui: docs/pages/blog/2021.md

  But we still had most of the work ahead of us, we had to roll out the improvements to all our users.
  With a large code and user base like MUI has, we knew it would take time.
  For example, we might have had migrated a few components to the new style engine and fix all the issues that we could find, going to production is unforgiving.
  Developers tend to surface all the cases that are overlooked in the first iteration with GitHub issues.
- On MUI X, we had successfully sold our first Pro plan licenses and reached 100k downloads/month on the MIT version.
  We had early signs from the community of a market fit for the product and that the open core model was the right one to execute our mission.
  We were determined to solely focus on the data grid component, and demonstrate that our model was viable to build the best React data grid in the space.

What might have been the biggest change in 2021 is the implications of a larger team for the organization structure.
We have reached an inflection point at around 10 people.
I could no longer look at all the GitHub issues and PRs.
At that point, it was requiring >70h/week to do it correctly.

> One individual continuing doing it was not scaling.

Reviewer: I didn't understand in the first time I read.

```suggestion
One individual doing that continuously was not scaling.
```

Author: I believe that your proposal changes the meaning. 

```suggestion
It was not scaling to have one individual aware of all PRs and GitHub issues.
```

> It would prevent growing the team, and more importantly, distract me off the core CEO responsibilities (people, vision, funding).

Author: ```suggestion
Continuing in this direction would prevent us to grow the team, and more importantly, it was distracting me from the core CEO responsibilities (people, vision, funding).
```

- We have broken down the demos into smaller and focus on one aspect at a time.
- We have polished and promoted most of the components that were in the lab to `@mui/material`.
- We have introduced many new components:
  - [Masonry](/components/masonry/)
  - [Stack](/components/stack/)
  - [Trap Focus](/components/trap-focus/)
  - [Unstyled button](/components/buttons/#unstyled)
  - [Unstyled slider](/components/slider/#unstyled)
  - [Unstyled modal](/components/modal/#unstyled)
  - [Unstyled slider](/components/switches/#unstyled)
  - [Unstyled tabs](/components/tabs/#unstyled)

### MUI X

- We have released [v5](/blog/mui-x-v5/), 42% of the npm downloads are on this major version now.
- We have introduced support for [custom locales](/components/data-grid/localization/).
- We have introduced many new features while consolidating the existing ones:
  - [Row editing](/components/data-grid/editing/#row-editing)
  - [Cell editing](/components/data-grid/editing/#cell-editing)
  - [Column pinning](/components/data-grid/columns/#column-pinning)
  - [Tree data](/components/data-grid/group-pivot/#tree-data)
  - [CSV export](/components/data-grid/export/#csv-export)

>   - [Print](/components/data-grid/export/#print)

Reviewer: ```suggestion
  - [Print export](/components/data-grid/export/#print)
```

Author: We reference it as only Print in the pricing table comparison. I would argue that Print is clear enough on its own.

- We have grown the MIT downloads on npm and the number of licenses sold per month by a factor of x10 in 2021 compared to 2020.
- We have grown the size of the team from 2 to 4 people (6 in January).

## Looking at 2022

I have been involved on MUI for 7 years and almost 3 years full-time now.
It's amazing to see the progress the library has done [since then](https://v0.mui.com/#/components/app-bar).
But at the same time, I'm astonished by how far we can still push the mission forward.
My experience has been that the more we improve the library, the more we envision future ways to do it better.
It's still [**day 1**](https://www.sec.gov/Archives/edgar/data/1018724/000119312517120198/d373368dex991.htm).

> What's our mission? We're making building great UIs more accessible.

Reviewer: "making building"?

An suggestion to keep it more explicit:

```suggestion
What's our mission? We're making the task of building great UIs more accessible.
```

Author: ```suggestion
What's our mission? To empower as many people as possible to build great UIs, faster.
```

Should we implement it later on for v6? Should we ignore it? Should we implement Google's UI that looks cleaner instead?
In the meantime, we will keep improving the components, increasingly relying on `@mui/base`.

#### MUI Joy

We will continue [in the direction of 2021](/blog/2020/#a-second-theme).
While Material Design is a predominant design choice for building rich client-side applications, it's not the only popular design trend.
We plan to provide a [second design system](https://github.com/mui-org/material-ui/issues/22485) to expand the range of use cases for MUI, striking a balance between:

- alignment with the best-looking design trends of hot startups in the US.
- optimization of its usage for rich client-side applications while keeping it good enough for landing pages.
- simplicity of the design for allowing simpler customizations.
- complementarity with Material Design.

It will be built on top of the unstyled components: MUI Base and our styling library: MUI System.

### MUI X

The data grid product initiative is working, it's time to scale it by growing the team and the product scope.

> We will keep the data grid as our primary effort, there are still [many features left to be built 🚧](/components/data-grid/getting-started/#feature-comparison) to get closer to the alternatives.

Reviewer: Does it make sense to create a separate title only for the DataGrid, like "MUI Material"? We could mention the opening of the apiRef on the MIT version, full documentation, hook-only API and the Premium plan.

> However, we will also expand the scope to the date picker components, resuming work on it.

Reviewer: ```suggestion
However, we will also expand the scope to the date picker components, resuming the work on it.
```
